rpc_util.go

```diff
@@ -159,6 +159,14 @@ type CallOption interface {
 	after(*callInfo)
 }
 
+// EmptyCallOption does not alter the Call configuration.
+// It can be embedded in another structure to carry satellite data for use
+// by interceptors.
+type EmptyCallOption struct{}
+
+func (EmptyCallOption) before(*callInfo) error { return nil }
+func (EmptyCallOption) after(*callInfo)        {}
```

Comment:
Can we add a 
`var _ CallOption = EmptyCallOption{}` to make sure this always satisfies the contract?